pages/examples/content/css-grid.md

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
I got this trick from webpack. Turns out you can write yaml in markdown and then parse it like I'm doing in the babel plugin to set the `meta` information 🎉